eos/b-decays/b-to-dstar-l-nu.cc

```diff
@@ -19,18 +19,183 @@
+
+                _vv[6] = NF * v * ( - (std::norm(a.a_para) - std::norm(a.a_perp)) + 16.0 * (std::norm(a.a_para_T) - std::norm(a.a_perp_T)) );
+
+                _vv[7] = NF * 2.0 * v * std::imag( a.a_para * std::conj(a.a_perp) );
+
+                _vv[8] = NF * std::sqrt(2.0) * v * std::real( a.a_para * std::conj(a.a_0) - 16.0 * a.a_para_T * std::conj(a.a_0_T) );
+
+                _vv[9] = NF * 2.0 * std::sqrt(2.0) * std::real( - a.a_perp * std::conj(a.a_0) + (1.0 - v) * ( a.a_para * std::conj(a.a_t_P) - 16.0 * a.a_perp_T * std::conj(a.a_0_T) ) + 4.0 * std::sqrt(1.0 - v) * ( a.a_0_T * std::conj(a.a_perp) + a.a_perp_T * std::conj(a.a_0) - a.a_para_T * std::conj(a.a_t_P) ) );
+
+                _vv[10] = NF * 2.0 * std::sqrt(2.0) * std::imag( - a.a_para * std::conj(a.a_0) + (1.0 - v) * a.a_perp * std::conj(a.a_t_P) + 4.0 * std::sqrt(1.0 - v) * ( a.a_0_T * std::conj(a.a_para) - a.a_para_T * std::conj(a.a_0) + a.a_perp_T * std::conj(a.a_t_P) ) );
+
+                _vv[11] = NF * std::sqrt(2.0) * v * std::imag( a.a_perp * std::conj(a.a_0) );
+            }
+
+            AngularObservables(const std::array<double, 12> & vv) :
+                _vv(vv)
+            {
+            }
+
+            inline double vv10()    const  { return _vv[0]; }
+            inline double vv20()    const  { return _vv[1]; }
+            inline double vv30()    const  { return _vv[2]; }
+            inline double vv1T()    const  { return _vv[3]; }
+            inline double vv2T()    const  { return _vv[4]; }
+            inline double vv3T()    const  { return _vv[5]; }
+            inline double vv4T()    const  { return _vv[6]; }
+            inline double vv5T()    const  { return _vv[7]; }
+            inline double vv10T()   const  { return _vv[8]; }
+            inline double vv20T()   const  { return _vv[9]; }
+            inline double vv30T()   const  { return _vv[10];}
+            inline double vv40T()   const  { return _vv[11];}
+
+            inline double v1()      const   { return _vv[0]; }
+            inline double v2()      const   { return _vv[1]; }
+            inline double v3()      const   { return _vv[2]; }
+            inline double v4()      const   { return _vv[3]; }
+            inline double v5()      const   { return _vv[4]; }
+            inline double v6()      const   { return _vv[5]; }
+            inline double v7()      const   { return _vv[6]; }
+            inline double v8()      const   { return _vv[7]; }
+            inline double v9()      const   { return _vv[8]; }
+            inline double v10()     const   { return _vv[9]; }
+            inline double v11()     const   { return _vv[10];}
+            inline double v12()     const   { return _vv[11];}
+
+            // longitudinal polarization amplitude
+            inline double normalized_amplitude_polarization_L() const
```

Comment:
Maybe do not call them "normalized_amplitude_polarization_L" - they do not seem real observables according to eq.(9) in 1405.3719. Please introduce instead 
    Gamma_L = 3/4 A_L
    Gamma_R = 3/4 A_T
the decay width for longitudinal and transversal polarized D^*